cpp/main.cpp

```diff
@@ -0,0 +1,45 @@
+#include <iostream>
```

Comment:
std::cout and std::cin, I'm not completely sure it was necessary, as I'm used to using Borland's C++ compiler, and the std namespace might be automatic in other compilers.